src/js/stores/wizard.js

```diff
@@ -0,0 +1,23 @@
+import {Store} from 'flummox';
+
+
+export default class WizardStore extends Store {
```

Comment:
It's correctly putting the store's data in localstorage, but it doesn't appear to be using that saved data when the store's lifecycle restarts.